mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/AddonController.java

```diff
@@ -220,7 +220,8 @@ private JSONArray buildAddonSpecs(String path) {
                 continue;
             }
             try {
-                JSONArray parsed = new JSONArray(FileUtils.readFileToString(curConfig));
+                String contents = FileUtils.readFileToString(curConfig, "UTF-8");
+				JSONArray parsed = new JSONArray(contents);
```

Comment:
This should make the linter unhappy (I agree, it should also have other way to copy an array in an array).